app/assets/javascripts/transitive_dependencies.js

```diff
@@ -0,0 +1,85 @@
+function ajaxDepOnClick(gem_id,ver_id) {
+  $.ajax({
+    type: "get",
+    url: "/gems/"+gem_id+"/versions/"+ver_id+"/transitive_dependencies.json",
+    success: function(resp) {
+      //var json_obj = JSON.parse(resp);
+      //alert(JSON.stringify(resp))
+      ajaxDepActions(resp)
+    }
+  });
+}
+
+function ajaxDepActions(resp) {
+  var gem_id = resp.rubygem_id
+
+  // when we have gems like http_parser.rb, we need to replace . with \. to access any ids with the name
+  gem_id = gem_id.replace(/\./, "\\.")
+
+  var all_deps = $('.deps_item').map(function() {
+    return $(this).attr('data-id');
+  }).get();
+
+  if (resp.run_deps.length != 0){
+    $("#"+gem_id+'runtime').find(".deps_scope").before("<span class='scope scope--expanded'>Runtime :</span>");
```

Comment:
Why are we reinserting here?

Reply:
That is where i'm inserting the name of scope (development or runtime)